packages/frontend/src/Build.svelte

```diff
@@ -703,15 +882,22 @@ $: if (availableArchitectures) {
                     <Button on:click={() => getBuildConfigFile()}>Browse...</Button>
                   </div>
                   <p class="text-sm text-[var(--pd-content-text)] pt-2">
-                    The build configuration file is a TOML or JSON file that contains the build options for the disk
-                    image. Customizations include user, password, SSH keys and kickstart files. More information can be
-                    found in the <Link
-                      externalRef="https://github.com/osbuild/bootc-image-builder?tab=readme-ov-file#-build-config"
-                      >bootc-image-builder documentation</Link
-                    >.
+                    This will override any above user-specific input and use the supplied file only.
```

Comment:
The text at the top only talks about filling in the fields, then at the bottom of the form you find out you can also provide a file. Feels like maybe there should be two radio buttons like 'Create new build config' and 'Supply existing build config file' so that it's clear you have either option from the start?